.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: php
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+php: 7.1
+
+before_install:
+  - phpenv config-rm xdebug.ini
+
+install:
+  - composer install
+
+before_script:
+  - mkdir -p .build/php-cs-fixer
+
+script:
+  - ./generate.sh
+  - git diff --exit-code
```

Comment:
Failing the build if there are any differences between the checked-in code and the result of running

```
$ ./generate.sh
```

Reply:
Sounds good!

Reply:
Glad you like it!